apps/site/pages/components/molecules/input-field.mdx

```diff
@@ -205,13 +205,19 @@ export const propsInputField = [
 #### Label
 
 <TokenTable>
-  <TokenRow token="--fs-input-label-padding" value="0 var(--fs-spacing-2)" />
+  <TokenRow token="--fs-input-field-label-padding" value="0 var(--fs-spacing-2)" />
```

Comment:
todo: We'll have to update in [developer portal doc](https://developers.vtex.com/docs/guides/faststore/molecules-input-field)

Reply:
@renatamottam since we migrate all the docs to portal, we'll need to apply all the changes made on this PR there.

cc: @Mariana-Caetano